repo/photos/photos.go

```diff
@@ -101,8 +95,11 @@ func Add(n *core.IpfsNode, pk libp2p.PubKey, p *os.File, t *os.File, lc string,
 	dirb := uio.NewDirectory(n.DAG)
 
 	// add the image
-	p.Seek(0, 0)
```

Comment:
i see, so you're re-reading the file below so you don't need to rewind this... do we need to read the file twice? looks like the file will be opened a total of 3 times, including what happens in the `ImagePathWithoutExif` method, do you think that makes sense?

Reply:
yeh, this is probably an issue. i could probably pass the reader directly to the `ImagePathWithoutExif` and `Seek` within there to read the exif data.